tests/test.nim

```diff
@@ -1,20 +1,39 @@
+import unittest
```

Comment:
Though the documentation for `unittest` suggests that `doAssert` is "easier", I don't know why it would put us off using what is clearly a superior testing method.

I switched to `unittest` to get counterexample printing, as `doAssert` just fails.